CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ x.x.x Release notes (yyyy-MM-dd)
 
 * Use reachability API to minimize the reconnection delay if the network
   connection was lost.
+* Add a new API for changing and retrieving permissions. This API can be used
```

Comment:
If the intention is for these APIs to replace the existing ones, we should use stronger wording:

```md
* Add a new API for changing and retrieving permissions for synchronized Realms.
  This API is intended to replace the existing Realm Object-based permissions
  system.
```